compair/static/less/assignment.less

```diff
@@ -22,18 +22,14 @@
     padding-top: .5em;
     color: @breadcrumb-gray;
     border-top: 2px dotted @green-blue;
+    overflow-x: hidden;
```

Comment:
What was overflowing? (haven't run the code yet so not sure what this is hiding)

Reply:
It's just a CSS "trick" to add separators between items that may span multiple lines without having the separator show at the beginning (where there should be no separators). This particular directive hides the separator at the beginning (along with a negative margin on the actual item).

Reply:
It just looks much cleaner this way compared to the way it was looking before.